modules/apps/portal-search/portal-search-rest-impl/rest-openapi.yaml

```diff
@@ -71,6 +71,13 @@ components:
             type: object
         SearchResult:
             properties:
+                actions:
```

Comment:
@marco-leo just checking:

1. the actions don't apply to search result entity but to the embedded one, so is the search result entity the right level to put the actions to or could they be inside the embedded entity?
1.  Is the idea to show actions always or only if `nestedFields=embedded`. 
1.  theoretically, what if there are multiple, nested embedded items, for example `user` having `contact` embedded or a like. Is the assumption that there can be only one set of actions and they only apply to the top level entity?